doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be able to be backward compatible with previous versions.
+6. Should be user friendly and intuitive even for people who are not IT savvy.
+7. Should have a fast response of within 2 seconds.
```

Comment:
Can be more specific. `Should have a fast response for frequently used commands of within 2 seconds under normal load.` (need to define what `normal load` separately)